docs/deployment/feature-gates.md

```diff
@@ -65,6 +67,7 @@ A *General Availability* (GA) feature is also referred to as a *stable* feature.
 
 ## List of Feature Gates
 
-| Feature          | Description                                                                                                                                                                                   |
-|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `UseEtcdWrapper` | Enables the use of etcd-wrapper image and a compatible version of etcd-backup-restore, along with component-specific configuration changes necessary for the usage of the etcd-wrapper image. |
+| Feature                        | Description                                                                                                                                                                                                                                                                                          |
+|--------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `UseEtcdWrapper`               | Enables the use of etcd-wrapper image and a compatible version of etcd-backup-restore, along with component-specific configuration changes necessary for the usage of the etcd-wrapper image.                                                                                                         |
+| `EnableSnapshotCompaction`     | Enables the snapshot compaction jobs in etcd-druid. When enabled, etcd-druid will periodically compact the snapshots to reduce storage usage.                                                                                                                                                       |
```

Comment:
```suggestion
| `EnableSnapshotCompaction`     | Enables the snapshot compaction controller in etcd-druid. When enabled, etcd-druid will periodically compact the snapshots to reduce storage usage.                                                                                                                                                       |
```
Maybe the wording needs to be changed as the controller does not create these jobs periodically, but on triggers